app/containers/InterventionDetailsPage/components/SessionListItem/index.js

```diff
@@ -231,13 +235,18 @@ function SessionListItem({
                     {index + 1}
                   </SessionIndex>
                   <Column px={15}>
-                    <Markup
-                      attributes={{
-                        dir: 'auto',
-                      }}
-                      content={formatSessionName(name)}
-                      tagName={H2}
-                    />
+                    <Row>
+                      <Markup
+                        attributes={{
+                          dir: 'auto',
+                        }}
+                        content={formatSessionName(name)}
+                        tagName={H2}
+                      />
+                      {type === SessionTypes.SMS_SESSION && (
+                        <Icon ml={8} src={mobileIcon} alt="sms" />
+                      )}
+                    </Row>
```

Comment:
For bidirectionality support do
```suggestion
                    <Row gap={8}>
                      <Markup
                        attributes={{
                          dir: 'auto',
                        }}
                        content={formatSessionName(name)}
                        tagName={H2}
                      />
                      {type === SessionTypes.SMS_SESSION && (
                        <Icon src={mobileIcon} alt="sms" />
                      )}
                    </Row>
```
or
```suggestion
                    <Row>
                      <Markup
                        attributes={{
                          dir: 'auto',
                        }}
                        content={formatSessionName(name)}
                        tagName={H2}
                      />
                      {type === SessionTypes.SMS_SESSION && (
                        <Icon paddingInlineStart={8} src={mobileIcon} alt="sms" />
                      )}
                    </Row>
```

Note: As there is a probability that CIAS will need to support bidirectionality in whole UI in the future, try using flex or logical properties in new changes to get support for RTL languages. This will reduce code changes required in the future if we would work on RTL support.